plenum/server/replica.py

```diff
@@ -453,7 +457,7 @@ def get_lowest_probable_prepared_certificate_in_view(self, view_no) -> Optional[
                 return n
         return None
 
-    def set_last_ordered_for_non_master(self):
+    def _set_last_ordered_for_non_master(self):
```

Comment:
I concern about the name, the function not only "sets" but also "clears" something. It can be unexpected behavioral for caller.